src/components/BounceRuleDetailed/index.stories.jsx

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { BrowserRouter } from "react-router-dom";
+import BounceRuleDetailed from ".";
+
+const sampleRule = {
+  id: 504,
+  response_code: "550",
+  enhanced_code: "",
+  regex: "no MX record for domain",
+  priority: 0,
+  description:
+    "bWFpbmx5IGxpYmVydHkgZG9tYWluIGJsb2NrIHNlZWluZyB+NTAlIG9mIGFkZHJlc3NlcyBlbmdhZ2luZyBTRyB3aWRl",
+  bounce_action: "no_action",
+};
+
+storiesOf("Individual Rule Details", module)
+  .add("Default", () => (
+    <BrowserRouter>
```

Comment:
Did we need to wrap this in BrowserRouter?

Reply:
I believe it was giving me some kind of error about not being able to have a Link without a wrapping Router

Reply:
![image](https://user-images.githubusercontent.com/26265748/49681656-90e92800-fa5a-11e8-94f6-6973ffd269e9.png)